AEP/aep-00001.md

```diff
@@ -0,0 +1,177 @@
+Once the check list items are implemented and merged, the AEP receives the Final state and the initial issue is closed as completed. When finalizing the AEP, links to relevant technical and/or user documentation can be added to the footnotes.
+
+A Draft AEP can change to the Active state, meaning that it will never be Final. This can be the case for Informational and Process AEPs. They can be either ammended to take new information into account, or be replaced by a superseding AEP, depending on the ArmoniK Development Team's and the initial author's choice.
+
+## AEP Rejection and Replacement
+
+When in a Draft or in an Accepted state, an AEP can be rejected (or withdrawn) if, after further consideration, the AEP shouldn't be accepted because of design flaws, issues discovered during the implementation process or if the proposal was a bad idea after all. This rejection can either be mandated by the ArmoniK Developer Team, or be due to a withdrawal by the author of the AEP (with the consent of the ArmoniK Developer Team). In both cases, the AEP changes to a Rejected state. The initial issue will be closed as "not completed", but may be reopened if the idea is deemed of importance and relevance again.
+
+If a subsequent AEP is made to supersede a Final or Active AEP, the superseded AEP changes to the Replaced state and its header must then contain the "Superseded-By" field.
+
+# What constitutes an AEP ?
+
+Each AEP should have the following parts/sections:
+
+1. **Preamble** – Fields in a table format containing meta-data about the AEP, including the AEP number, a short descriptive title, the names, and optionally the contact info for each author, etc. The table is preceded by a top-level heading with the AEP number and short title.
+
+2. **Abstract** – a short description of the technical issue being addressed.
+
+3. **Motivation** – The motivation is critical for AEPs that want to change ArmoniK's functionalities, ecosystem or processes. It should clearly explain why the existing specification is inadequate to address the problem that the AEP solves. This can include collecting documented support for the AEP from important projects using ArmoniK or belonging to the ArmoniK ecosystem. AEP submissions without sufficient motivation may be rejected.
+
+4. **Rationale** – The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other softwares.  
+The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
+
+5. **Specification** – The technical specification should describe the syntax, semantics and functionning of any new ArmoniK feature. The specification should be detailed enough to allow proper implementations for at least the current major languages used supported by ArmoniK.Api (C#, C++, Python).
```

Comment:
```suggestion
5. **Specification** – The technical specification should describe the syntax, semantics and functionning of any new ArmoniK feature. The specification should be detailed enough to allow proper implementations for at least the current major languages used supported by ArmoniK.Api (C#, C++, Python, Javascript...).
```

Reply:
Technically it's Typescript which is missing iirc